RELEASES.md

```diff
@@ -1,3 +1,266 @@
+Version 1.32.0 (2019-01-17)
+==========================
+
+Language
+--------
+#### 2018 edition
+- [You can now use the `?` operator in macro definitions.][56245] The `?`
+  operator allows you to specify zero or one repetitions similar to the `*` and
+  `+` operators.
+
+#### All editions
+- [You can now match against `PhantomData<T>` types.][55837]
+- [You can now match against literals in macros with the `literal`
+  specifier.][56072] This will match against a literal of any type.
+  E.g. `1`, `'A'`, `"Hello World"`
+- [Self can now be used as a constructor and pattern.][56365] E.g. 
+  ```rust
+  struct Point(i32, i32);
+
+  impl Point {
+      pub fn new(x: i32, y: i32) -> Self {
+          Self(x, y)
+      }
+
+      pub fn is_origin(&self) -> bool {
+          match self {
+              Self(0, 0) => true,
+              _ => false,
+          }
+      }
+  }
+  ```
+- [Self can also now be used in type definitions.][56366] E.g.
+  ```rust
+  enum List<T>
+  where
+  Self: PartialOrd<Self> // can write `Self` instead of `List<T>`
+  {
+    Nil,
+    Cons(T, Box<Self>) // likewise here
+  }
+  ```
```

Comment:
+ Will need to add `uniform_paths` stabilization, https://github.com/rust-lang/rust/pull/56759

Reply:
Scratch that. We decided to not backport.

Reply:
Scratch the scratch; We decided to backport.